libs/init_runtime/include/hpx/hpx_init_params.hpp

```diff
@@ -51,6 +51,17 @@ namespace hpx {
     }    // namespace resource
     /// \endcond
 
+    enum runtime_type
+    {
+        runtime_type_local,
+        runtime_type_distributed,
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
+        runtime_type_best = distributed
+#else
+        runtime_type_best = local
+#endif
```

Comment:
Should name this `..._default` instead?

Reply:
Caught me. I had it as a scoped enum and used `best` since `default` won't work there. Then I changed it to an unscoped one to be consistent with the other enums, and so that I could use default. I missed this spot though. I'm currently lacking a good replacement for `default`.

Reply:
`default_`?

Reply:
In any case, whatever we do here we should do for the other enums as well.

Reply:
Absolutely. I don't like adding underscores just to avoid name conflicts, but it may be better than  any alternative name I'll come up with. I'll sleep on it and see if I can come up with something else but otherwise it can be `default_`.

Reply:
I wouldn't expect anybody using this value explicitly. This would be mostly for use as a default argument value on our API declarations.

Reply:
For the record, we discussed this here: #4127